src/aggregator/server/m3msg/server.go

```diff
@@ -24,6 +24,8 @@ import (
 	"fmt"
 	"io"
 
+	"github.com/uber-go/tally"
```

Comment:
nit: Heh, so until we get the linter would you mind grouping this with the other third party import groups (the last group in this file).